edgelet/Cargo.toml

```diff
@@ -14,6 +14,16 @@ members = [
     "support-bundle",
 ]
 
+[patch.'https://github.com/Azure/iot-identity-service']
+aziotctl-common = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
+aziot-certd-config = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
+aziot-identity-common = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
+aziot-identityd-config = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
+aziot-key-common = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
+aziot-keyd-config = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
+aziot-keys-common = { git = 'https://github.com/alexclewontin/iot-identity-service', branch = 'feature/snapping-lewontin' }
```

Comment:
When https://github.com/Azure/iot-identity-service/pull/520 lands, these can get reduced to point to the `feature/snapping` branch of Azure/iot-identity-service, instead of my personal branch in my personal repo